dashboards-reports/server/utils/validationHelper.ts

```diff
@@ -36,12 +36,15 @@ import {
 import { REPORT_TYPE } from '../../server/routes/utils/constants';
 
 export const isValidRelativeUrl = (relativeUrl: string) => {
-  const normalizedRelativeUrl = path.posix.normalize(relativeUrl);
+  let normalizedRelativeUrl = relativeUrl
+  if (!relativeUrl.includes('notebooks-dashboards')) {
```

Comment:
Why not normailze for notebook url?

Reply:
If we normalize for notebook url then the `?view=output_only` parameter will be erased and it will fail the RegEx validation for base URL

Reply:
can you add UT for this?

Reply:
Sure- to test what? That the notebook URL remains untouched?

Reply:
I think so